tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from page_objects.connections_page import ConnectionsPage
+from page_objects.login_page import LoginPage
+from utilities.config_reader import ReadConfig
+from utilities.driver_factory import create_driver_factory
+
+
+@pytest.fixture()
+def create_driver():
+    driver = create_driver_factory(ReadConfig.get_browser_id())
+    driver.maximize_window()
+    driver.get(ReadConfig.get_app_base_url())
+    yield driver
+    driver.quit()
+
+
+@pytest.fixture()
+def open_login_page(create_driver):
+    return LoginPage(create_driver)
+
+
+@pytest.fixture()
+def login_to_app(open_login_page):
+    return open_login_page.login(ReadConfig.get_user_creds())
+
+
+@pytest.fixture()
+def go_to_connections_page(open_login_page, create_driver):
+    open_login_page.login(ReadConfig.get_user_creds())
+    return ConnectionsPage(create_driver)
```

Comment:
login return SideBar? 
Cant get logic. How it can be that same action redirect you to SideBar and ConnectionsPage

Reply:
Sidebar is the shared component that is available from any page. And since I'm checking login via element on Sidebar (Login > Logout buttons), I decided the separate it, and do not place it at ConnectionsPage, which is the first you see after logining to the app. Moreover, the further usage of navigation made from SideBar does not involve ConnectionPage, so for me it looks reasonable.